presto-main/src/main/java/com/facebook/presto/sql/gen/BytecodeExpressionVisitor.java

```diff
@@ -79,6 +91,9 @@ public BytecodeNode visitCall(CallExpression call, final Scope scope)
                     // (SWITCH <expr> (WHEN <expr> <expr>) (WHEN <expr> <expr>) <expr>)
                     generator = new SwitchCodeGenerator();
                     break;
+                case TRY:
+                    generator = new TryCodeGenerator(call, tryExpressionsMap, inputParameters);
```

Comment:
I believe you don't have to pass in `call` here if you remove the top level try call for every key of `tryExpressionsMap`.